### ISSUE TITLE
Unable to switch provider on 12.1

### ISSUE THREAD

**Issue by Octelly** [skittles9823/QuickSwitch#42]
![Screenshot_20231017-150346_QuickSwitch.png](https://github.com/skittles9823/QuickSwitch/assets/29976879/a2f71f78-58d4-4f7a-8258-dccc3c980c0d)

![Screenshot_20231017-150500_Settings.png](https://github.com/skittles9823/QuickSwitch/assets/29976879/6f31934e-f256-4da5-8e65-06e9327f9f6e)

I can see the module installed in Magisk, but it's just stuck on Pixel Launcher no matter how many times I try to change it.

![Screenshot_20231017-150635_Lawnchair.png](https://github.com/skittles9823/QuickSwitch/assets/29976879/8634f717-6955-4785-a53c-e5584fac1d02)

The device is Samsung Galaxy A40

**Comment by tsaiajie** [comment]
This isn't a QuickSwitch problem.  
Lawnchair 12.1 is not compatible with QuickSwitch on Android 11 and 12.   
Make sure you are using Android 12L before installing Lawnchair 12.1!

**Comment by Octelly** [comment]
@tsaiajie yeah, I saw the other issue talking about this before creating this one. I'm pretty sure I'm using the correct version. I mean, I tried both anyway...
